sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 - Move DataLake SAS into `Azure::Storage::Sas` namespace.
 - `EncrytionKeySha256` are changed to binary(`std::vector<uint8_t>`).
+- Replaced all transactional content MD5/CRC64 with `ContentHash` struct.
+- `DataLakeHttpHeaders` is renamed to `PathHttpHeaders`, and now contains `ContentHash` for the resource.
```

Comment:
`PathHttpHeaders`? This doesn't sound relevant to this PR.

Reply:
This is relevant for this RP due to alignment to .NET SDK for HttpHeaders, including the name of the HttpHeaders and it's content.

Reply:
Fine